### DISCUSSION TITLE
Using MinIO with log aggregate on high traffic apps

### DISCUSSION THREAD

**Discussion by a-h-abid** [minio/minio#13938] [Q&A]
I just have a small query.

Is using MinIO with a log aggregator like fluentd on a high traffic app have any major impact on MinIO server??

https://docs.min.io/docs/aggregate-apache-logs-with-fluentd-and-minio.html

**Comment by harshavardhana** [comment]
> Is using MinIO with a log aggregator like fluentd on a high traffic app have any major impact on MinIO server??

No as long as hardware is capable
